client/src/components/InputBox/InputBox.js

```diff
@@ -9,17 +9,28 @@ export const InputBox = ({
   onChange,
   value,
   expanded,
+  multiline,
 }) => {
   return (
     <>
       <p className={styles.inputHeader}>{header}</p>
-      <input
-        className={`${styles.inputBox} ${expanded ? styles.description : ""}`}
-        type={type}
-        placeholder={placeholder}
-        onChange={(e) => onChange(e)}
-        value={value}
-      />
+      {multiline ? (
+        <textarea
```

Comment:
Minor design comment: Is there a way that perhaps the scroll bar could fit inside the actual box rather than overlapping it?
![image](https://user-images.githubusercontent.com/24628243/163659395-0d51faa4-806a-48f4-a087-bf552052a50c.png)

Reply:
Note for future design component implementations: Generally for this Description InputBox, we should probably have a separate component for input boxes that require larger text areas but for our use case this will suffice

Reply:
Couldn't find a proper fix for this. I could only get it to show a tiny box that is barely visible.

Reply:
That's ok for our use case since we'll be focusing more on functionality than aesthetics